force-app/main/default/objects/ServiceDelivery__c/fieldSets/Attendance_Service_Deliveries.fieldSet-meta.xml

```diff
@@ -26,6 +26,16 @@
         <isFieldManaged>false</isFieldManaged>
         <isRequired>false</isRequired>
     </availableFields>
+    <availableFields>
+        <field>LastModifiedById</field>
+        <isFieldManaged>false</isFieldManaged>
+        <isRequired>false</isRequired>
+    </availableFields>
+    <availableFields>
+        <field>LastModifiedDate</field>
+        <isFieldManaged>false</isFieldManaged>
+        <isRequired>false</isRequired>
+    </availableFields>
```

Comment:
We expose this info in the component anyway, but I guess it doesn't hurt as long as r-e-f handles them as read-only.